cmd/kots/cli/plugin.go

```diff
@@ -0,0 +1,201 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/pkg/kotsutil"
+	"github.com/replicatedhq/kots/pkg/logger"
+	"github.com/replicatedhq/kots/pkg/preflight"
+	"github.com/replicatedhq/kots/pkg/template"
+	analyzerunner "github.com/replicatedhq/troubleshoot/pkg/analyze"
+	"github.com/replicatedhq/troubleshoot/pkg/oci"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func PluginCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "plugin",
+		Short:         "Run commands for helm plugin",
+		Long:          `Run commands for helm plugin, this serves as an entry point for subcommands such as preflight`,
+		SilenceUsage:  false,
+		SilenceErrors: false,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			viper.BindPFlags(cmd.Flags())
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				cmd.Help()
+				os.Exit(1)
+			}
+			log := logger.NewCLILogger(cmd.OutOrStdout())
+
+			if args[0] == "preflight" {
```

Comment:
These should be defied as subcommands they they will show up as subcommands in help output and not as flags as dined below (because these are not flags)

Example of subcommands for `admin-console` [here](https://github.com/replicatedhq/kots/blob/main/cmd/kots/cli/admin-console.go#L100-L103)

Reply:
done